db/schema.rb

```diff
@@ -129,8 +129,8 @@
     t.string   "name"
     t.string   "surname"
     t.string   "email"
-    t.boolean  "mailing_list_consent",        default: false
-    t.string   "token",                                       null: false
+    t.boolean  "mailing_list_consent", default: false
+    t.string   "token", null: false
```

Comment:
This was failing the rake tests again, so had to update.

Once this PR or [this one](https://github.com/codebar/planner/pull/1435) are merged in, I will no longer need to update this.